jetty-jaas/src/test/java/org/eclipse/jetty/jaas/JAASLdapLoginServiceTest.java

```diff
@@ -41,6 +43,7 @@
 /**
  * JAASLdapLoginServiceTest
  */
+@EnabledForJreRange(max = JRE.JAVA_17, disabledReason = "sun.security.x509.X509CertInfo.set not present in Java 21, needs a Java 21 compatible version of Apache Directory Server")
```

Comment:
Why? This works.

Reply:
Why? This works fine.

Reply:
It didn't work.

See https://jenkins.webtide.net/blue/organizations/jenkins/jetty.project/detail/PR-10607/8/tests

This is because this test is using `@RunWith(FrameworkRunner.class)` which is based on Junit 4, which means that Runner isn't even looking at the Junit 5 annotations like `@EnabledForJreRange`.

Reply:
I added commit https://github.com/eclipse/jetty.project/pull/10607/commits/1dce158e04b20b7bee1403e5cc5f796a717a2ed9

which makes this specific test excluded on JDK 21+, which will make command line builds and CI builds happy (but not IDE builds)